cdap-docs/admin-manual/source/upgrading/cloudera.rst

```diff
@@ -30,7 +32,8 @@ in the Cloudera Manager UI.
 
 Upgrading CDAP Major/Minor Release Versions
 -------------------------------------------
-Upgrading between major versions of CDAP involves the additional steps of upgrading the
+Upgrading between major versions of CDAP (for example, from a 3.3.x version to |short-version|\.x)
```

Comment:
instead of hardcoding ``3.3.x``, can we use your ``|previous-short-version|\.x`` here?